clients/scala/src/main/scala/Runner.scala

```diff
@@ -0,0 +1,43 @@
+import java.io.{BufferedInputStream, BufferedOutputStream}
+import java.net.Socket
+
+import model.PlayerMessageGame.ActionMessage
+import util.StreamUtil
+
+object Runner extends App {
+
+  val host = if (args.length < 1) "127.0.0.1" else args(0)
+  val port = if (args.length < 2) 31001 else args(1).toInt
+  val token = if (args.length < 3) "0000000000000000" else args(2)
+
+  run(host, port, token)
+
+  def run(host: String, port: Int, token: String) {
+    val socket = new Socket(host, port)
+    socket.setTcpNoDelay(true)
+    val inputStream = new BufferedInputStream(socket.getInputStream)
```

Comment:
Don't you need to make sure `inputStream ` and `outputStream ` are closed properly in case of failure? `try finally`

Reply:
That is a good point, but these instances are the only for the whole JVM life, so i guess it does not really matter here.